source/Playlist.h

```diff
@@ -0,0 +1,64 @@
+/* Playlist.h
+Copyright (c) 2022 by RisingLeaf
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "ConditionSet.h"
+#include "LocationFilter.h"
+#include "Track.h"
+#include "WeightedList.h"
+
+
+#include <string>
+#include <map>
+#include <vector>
+
+
+class DataNode;
+class PlayerInfo;
+
+
+// Class to store a track of music that can be used in a playlist.
```

Comment:
Not sure but I think we're supposed to have one more whitespace (also isnt the { nelow supposed to be at tje same line as the class?